test/ex_json_schema/validator_test.exs

```diff
@@ -667,10 +667,14 @@ defmodule ExJsonSchema.ValidatorTest do
     assert :ok == validate(%{"format" => "date-time"}, false)
   end
 
+  test "format validation succeeds for datetime with tz offset" do
+    assert :ok == validate(%{"format" => "date-time"}, "2012-12-12 12:12:12Z")
```

Comment:
Shouldn't this test rather be that it works without a timezone? I think the different ways of specifying the time zones are already covered by the official JSON Schema test suite.

Reply:
Maybe I could add that other test case, I thought it would be important not to break the backward compat with my changes, but it makes sense to cover both, with and without timezone.

Reply:
I think it's safe to rely on the very comprehensive test suite for the dates with time zone: https://github.com/json-schema-org/JSON-Schema-Test-Suite/blob/689d2f28182720d3778ca91c149f725a6ce73f71/tests/draft4/optional/format.json
The only thing it doesn't cover apparently is date-times with no time zone, which are valid according to ISO 8601.